README.rst

```diff
@@ -15,15 +15,14 @@ Supply host-specific variables in ``group_vars/$hostname``.
 Bootstrapping
 -------------
 
-To bootstrap a newly-installed system, use ``./bootstrap HOSTNAME``.
+To bootstrap a newly-installed system, use ``./bootstrap``.
```

Comment:
This should now be run on the target host?  That will be worth noting.

Reply:
..and in a checked-out copy of the repo

Reply:
I feel the first is noted by the "Run as root" requirement, but there's nothing about needing the repository yet.